examples/example-vite-grpc-playground/src/main.tsx

```
@@ -0,0 +1,31 @@
+            relayUrl: dojoConfig.relayUrl,
+            worldAddress: dojoConfig.manifest.world.address,
+        },
+        domain: {
+            name: "WORLD_NAME",
+            version: "1.0",
+            chainId: "KATANA",
+            revision: "1",
+        },
+    });
+
+    createRoot(document.getElementById("root")!).render(
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)